challenges/test/lib/src/test/java/code/treetest/TreeTest.java

```diff
@@ -169,4 +169,40 @@ public void binarySearchTreeTest(){
 
     assertEquals(1234,newTree.findMaximunValue());
   }
+
+  @Test
+  public void testBreathFirstBinaryTree(){
```

Comment:
Great work with your test. You can always create one or two trees, then utilize them for all of your tests.